File: Modificador de 4 lenguaje/js/claseinterfaz/Guitarra.js
const IInstrumento = require("./IInstrumento");

class Guitarra extends IInstrumento {
    constructor(marca) {
        super();
        this.marca = marca;
    }

    tocar() {
        return `La guitarra ${this.marca} está sonando `;
    }
}

module.exports = Guitarra;

File: Modificador de 4 lenguaje/C#/ClasesAbstractas/ClasesAbstractas/Guitarra.cs
namespace EjemploConstructores.ClasesAbstractas
{
    class Guitarra : Instrumento
    {
        // Constructor con parámetros
        public Guitarra(string nombre) { Nombre = nombre; }

        // Constructor de copia
        public Guitarra(Guitarra otro) { Nombre = otro.Nombre; }

        public override void Tocar() => System.Console.WriteLine($"{Nombre} está sonando.");
    }
}
